edge-modules/TwinTester/Program.cs

```diff
@@ -0,0 +1,56 @@
+// Copyright (c) Microsoft. All rights reserved.
+namespace TwinTester
+{
+    using System;
+    using System.Threading;
+    using System.Threading.Tasks;
+    using Microsoft.Azure.Devices;
+    using Microsoft.Azure.Devices.Client;
+    using Microsoft.Azure.Devices.Edge.ModuleUtil;
+    using Microsoft.Azure.Devices.Edge.Util;
+    using Microsoft.Extensions.Logging;
+
+    class Program
+    {
+        static readonly ILogger Logger = ModuleUtil.CreateLogger("TwinTester");
+
+        static async Task Main()
+        {
+            Logger.LogInformation($"Starting twin tester with the following settings:\r\n{Settings.Current}");
+
+            try
+            {
+                RegistryManager registryManager = RegistryManager.CreateFromConnectionString(Settings.Current.ServiceClientConnectionString);
+
+                ModuleClient moduleClient = await ModuleUtil.CreateModuleClientAsync(
+                    Settings.Current.TransportType,
+                    ModuleUtil.DefaultTimeoutErrorDetectionStrategy,
+                    ModuleUtil.DefaultTransientRetryStrategy,
+                    Logger);
+                await moduleClient.OpenAsync();
+
+                AnalyzerClient analyzerClient = new AnalyzerClient { BaseUrl = Settings.Current.AnalyzerUrl.ToString() };
+
+                Storage storage = new Storage();
+                storage.Init(Settings.Current.StoragePath, new SystemEnvironment(), Settings.Current.StorageOptimizeForPerformance);
+
+                TwinState twinState = await TwinOperator.InitializeModuleTwin(registryManager, moduleClient, storage);
+                TwinOperator twinOperator = new TwinOperator(registryManager, moduleClient, analyzerClient, storage, twinState);
+
+                TimeSpan validationInterval = new TimeSpan(Settings.Current.TwinUpdateFailureThreshold.Ticks / 4);
+                PeriodicTask periodicValidation = new PeriodicTask(twinOperator.PerformValidation, validationInterval, validationInterval, Logger, "TwinValidation");
+                PeriodicTask periodicUpdate = new PeriodicTask(twinOperator.PerformUpdates, Settings.Current.TwinUpdateFrequency, Settings.Current.TwinUpdateFrequency, Logger, "TwinUpdates");
```

Comment:
Varun made a comment in Dylan's load gen PR about wrapping the task with the cancellation token from Shutdown Handler. Can someone elaborate more? Not sure how to do this.